test/com/amazon/ion/IonTestCase.java

```diff
@@ -231,7 +259,14 @@ public IonDatagram load(File ionFile)
         throws IonException, IOException
     {
         IonLoader loader = loader();
-        IonDatagram dg = loader.load(ionFile);
+        IonDatagram dg;
+        if (ionFile.getName().endsWith(".ion")) {
+            dg = loader.load(ionFile);
+        } else {
+            IonReader reader = myStreamingMode.newIonReader(system().getCatalog(), new FileInputStream(ionFile));
+            dg = loader.load(reader);
+            reader.close();
+        }
```

Comment:
Yes, that's why. This can be removed and replaced with `dg = loader.load(ionFile)` once #379 is complete. I've added a comment to that effect.

I added the check for the `.ion` prefix for now to catch the text Ion test files and change as little as possible about how those are tested compared to before. This isn't a requirement.